internal/templates/data/common/py/boot.py

```diff
@@ -4,6 +4,6 @@
 if __name__ == "__main__":
     logging.basicConfig(
         level=logging.INFO,
-        format='[%(levelname).5s] %(message)s',
+        format='[%(levelname)5.5s] %(message)s',
```

Comment:
left align the levelname but put the extra spacing on the right before the bracket